jlibra-core/src/main/java/dev/jlibra/admissioncontrol/transaction/U64Argument.java

```diff
@@ -1,8 +1,8 @@
 package dev.jlibra.admissioncontrol.transaction;
 
-import dev.jlibra.serialization.ByteSequence;
-import dev.jlibra.serialization.Serializer;
+import dev.jlibra.serialization.lcs.LCS;
 
+@LCS.ExternallyTaggedEnumeration(dev.jlibra.serialization.lcs.type.TransactionArgument.U64)
```

Comment:
What do you think about removing `LCS` wrapper?

Reply:
There are now three different annotations related to the serialization and I thought it would make it clearer if they have this LCS namespace, I think its many times a problem when using annotations that it's not that clear what they are used for (like when using spring framework, jpa and jax rs annotations in same classes)
I'd like to have the context somehow visible in the annotation name, either by this way or prefixing the annotation name with LCS (LCSField, LCSExternallyTaggedEnumeration..),